src/types.ts

```diff
@@ -0,0 +1,422 @@
+import { BN } from "@project-serum/anchor";
+import { Cluster, PublicKey } from "@solana/web3.js";
+import Decimal from "decimal.js";
+
+
+
+export type LiquidatorAssociatedTokenAccounts = {
+    [key in CollateralTokenActive]: string;
+};
+
+export type Loan = {
+    metadata: UserMetadata,
+    usdhDebt: BN | Decimal
+    collateral: CollateralAmounts,
+    tcr: BN | Decimal
```

Comment:
tcr stands for TotalCollateralRatio and refers to the entire System, whereas for a user it's typically ICR (individual collateral ratio)

Reply:
Okay will make that distinction